aip-console-tools-core/src/main/java/com/castsoftware/aip/console/tools/core/services/UploadServiceImpl.java

```diff
@@ -83,8 +82,9 @@ public String uploadFileForOnboarding(File filePath, String applicationGuid) thr
         String sourcePath;
         final String[] uploadedFilePath = new String[1];
         String archiveExtension = com.castsoftware.aip.console.tools.core.utils.FilenameUtils.getFileExtension(filePath.getName());
+        String fileName = com.castsoftware.aip.console.tools.core.utils.FilenameUtils.getFileName(filePath.getName());
```

Comment:
Please also check when filename passed as an environment variable

Reply:
filename is not present in env variable. only file path can be provided.